flink-java/src/main/java/org/apache/flink/api/java/functions/GroupReduceIterator.java

```diff
@@ -24,6 +24,11 @@
 
 import java.util.Iterator;
 
+/**
+ * Base class that simplifies reducing all values provided as {@link Iterable}.
```

Comment:
-> "A {@link RichGroupReduceFunction}" that returns an {@link Iterable} instead."